config.xml

```diff
@@ -11,36 +11,46 @@
         <package name="7zip-15-05.vm"/>
         <package name="apimonitor.vm"/>
         <package name="apktool.vm"/>
+        <package name="bytecodeviewer.vm"/>
         <package name="capa.vm"/>
         <package name="cmder.vm"/>
+        <package name="common.vm"/>
         <package name="cyberchef.vm"/>
         <package name="cygwin.vm"/>
+        <package name="de4dot-cex.vm"/>
+        <package name="dependencywalker.vm"/>
         <package name="die.vm"/>
         <package name="dnspyex.vm"/>
         <package name="explorersuite.vm"/>
         <package name="fakenet-ng.vm"/>
         <package name="floss.vm"/>
         <package name="ghidra.vm"/>
         <package name="hashmyfiles.vm"/>
+        <package name="hollowshunter.vm"/>
+        <package name="hxd.vm"/>
         <package name="idafree.vm"/>
         <package name="libraries.python3.vm"/>
         <package name="map.vm"/>
         <package name="notepadplusplus.vm"/>
         <package name="notepadpp.plugin.compare.vm"/>
-        <package name="ollydbg.ollydumpex.vm"/>
-        <package name="ollydbg.vm"/>
-        <package name="ollydbg2.ollydumpex.vm"/>
-        <package name="ollydbg2.vm"/>
+        <package name="npcap.vm"/>
```

Comment:
dependency of wireshark

Reply:
wireshark will already install it (as it is as dependency in the nuspec). So I think we don't need to explicitly add it.